gazebo_plugins/src/gazebo_ros_wheel_slip.cpp

```diff
@@ -75,25 +89,63 @@ void GazeboRosWheelSlip::Load(gazebo::physics::ModelPtr _model, sdf::ElementPtr
     [this](std::vector<rclcpp::Parameter> parameters) {
       auto result = rcl_interfaces::msg::SetParametersResult();
       result.successful = true;
+
       for (const auto & parameter : parameters) {
         std::string param_name = parameter.get_name();
+
+        if (param_name.find("slip_compliance_unitless_lateral/") != std::string::npos) {
+          auto wheel_name = param_name.substr(param_name.find("/") + 1);
+          if (impl_->mapSlipLateralDefault.count(wheel_name) == 1) {
+            // The parameter has a valid name
+            double slip = parameter.as_double();
+            if (slip >= 0.) {
+              RCLCPP_INFO(
+                impl_->ros_node_->get_logger(),
+                "New lateral slip compliance for %s: %.3e", wheel_name.c_str(), slip);
+              this->SetSlipComplianceLateral(wheel_name, slip);
```

Comment:
Since this parameters callback is meant for validation, I suggest we do the actual setting of slip compliance values in the ParameterEventHandler callback instead. Consider using [this method](https://github.com/ros2/rclcpp/blob/d5ec2580805c3abd8b6fecaf3d09040e00e9f15e/rclcpp/include/rclcpp/parameter_event_handler.hpp#L192-L194) to subscribe to all parameter events and filter inside the callback like we are doing here.